### ISSUE TITLE
Relase new version

### ISSUE THREAD

**Issue by LLFourn** [bitcoindevkit/rust-esplora-client#32]
Thanks!

**Comment by rajarshimaitra** [comment]
I was about open the same issue for some other unrelated bug I observed while working with bdk's `EsploraBlockchain`. So I would just go ahead and document it here..

While updating `electrsd` to `v0.22` in bdk, `ureq` resolution doesn't happen and build fails with the following.

```
Updating crates.io index
error: failed to select a version for `ureq`.
    ... required by package `bitcoind v0.28.0`
    ... which satisfies dependency `bitcoind = "^0.28.0"` of package `electrsd v0.22.0`
    ... which satisfies dependency `electrsd = "^0.22"` of package `bdk v0.26.0 (/home/raj/github-repo/bdk)`
versions that meet the requirements `^2.5.0` are: 2.5.0

all possible versions conflict with previously selected packages.

  previously selected package `ureq v2.2.0`
    ... which satisfies dependency `ureq = "~2.2.0"` of package `esplora-client v0.2.0`
    ... which satisfies dependency `esplora-client = "^0.2"` of package `bdk v0.26.0 (/home/raj/github-repo/bdk)`
```

This is happening because the `ureq` dep of this crate has been updated to `2.5.0` from `~2.2.0` in this commit, https://github.com/bitcoindevkit/rust-esplora-client/commit/cd1cc306f3c079f8f26b225b8c7693183fcfe9fe, which was done after the `0.2.0` release of this crate. 

So would be helpful to have another release.. 

So yes please.. 🙂

**Comment by vladimirfomene** [comment]
Closing this as we just released version 0.3.0 of the software.
